tests/test-module.py

```diff
@@ -90,44 +90,6 @@ def test_load_and_update_remote_event_does_not_overwrite_configuration(self):
                     'replace': self.config_event_with_replace['cma']['event']['replace']}
         self.assertEqual(expected, result)
 
-    # load_nested_event tests
-    def test_returns_load_nested_event_local(self):
```

Comment:
This test is no longer needed as we've removed this logic (we don't care about the task name)